lib/Service/LockService.php

```diff
@@ -388,4 +389,20 @@ private function propagateEtag(LockContext $lockContext): void {
 		]);
 		$node->getStorage()->getUpdater()->propagate($node->getInternalPath(), $node->getMTime());
 	}
+
+	/**
+	 * @param string $message
+	 * @param bool $trace
+	 * @param array $serializable
+	 */
+	private function notice(string $message, bool $trace = false, array $serializable = []) {
+		$opts = ['app' => 'files_lock'];
+		if ($trace) {
+			$opts['exception'] = new HintException($message, json_encode($serializable));
+		} elseif (!empty($serializable)) {
+			$message .= ' -- ' . json_encode($serializable);
+		}
+		$this->logger->notice('[notice] ' . $message, $opts);
+	}
+
```

Comment:
Mostly took this from the `TLogger` trait - but i don't know how the `Psr\Log\LoggerInteface` works:
* do we still need to prefix the message with `[notice] `?
* do the $opts have the same effect they used to have in the `TLogger`?